scipy/stats/tests/test_mstats_basic.py

```diff
@@ -550,6 +550,20 @@ def test_trimmedmean(self):
         assert_almost_equal(mstats.trimmed_mean(data,(0.1,0.1)), 343, 0)
         assert_almost_equal(mstats.trimmed_mean(data,(0.2,0.2)), 283, 0)
 
+    def test_trimmedvar(self):
+        data = ma.array([77, 87, 88,114,151,210,219,246,253,262,
+                         296,299,306,376,428,515,666,1310,2611])
+        assert_almost_equal(mstats.trimmed_var(data,0.1), 79989.9723, 4)
+        assert_almost_equal(mstats.trimmed_var(data,0.1,ddof=1), 84989.3456, 4)
+        assert_almost_equal(mstats.trimmed_var(data,(0.2,0.2)), 10966.9822, 4)
```

Comment:
@tupui should the linter not be failing here? Should there not be whitespace after the commas?

Reply:
Sadly no the "linter" at the moment is more a syntax check that anything else. Ruff is optimally used in combination with Black... Ruff does not re-implement all stylistic only rules. At the moment we have some miss due to that yes.

Reply:
@vmpyr, what this means is that, at the moment, our CI does not automatically check for [PEP 8](https://peps.python.org/pep-0008/) code style violations, so it will be up to contributors and reviewers to ensure that the code conforms to the style guidelines (well, that is *always* true, but at the moment we have fewer tools helping us out).  In this case, PEP 8 says that there should be a space after a comma.  Note that when you fix line 557, the line length will probably exceed 79 characters, so you'll have to wrap that line.

Reply:
@WarrenWeckesser the current `lint.toml` file is configured to line length 88 but I wasn't sure if this was a conscious change

Reply:
If the line length is configured to length 88 then it will fit even after the space

Reply:
By the way, there isn't a space after a comma in the surrounding code, which is why I did not put one as well

Reply:
Surrounding code is not always a reference. SciPy is more than 20 years old 😉 

Not sure about why 88 is there. Smells like Black, even if should be 79 and 88 is there to allow some needed relaxation for things like URL.

Reply:
Yes, I should've known better, the surrounding code is 15 years old, I just checked xD